themes/socialbase/templates/file/image-widget.html.twig

```diff
@@ -43,7 +43,7 @@
     <div class="hidden">{{ data.remove_button }}</div>
     <button type="button" id="post-photo-remove" class="btn--post-remove-image">
       <svg class="btn-icon">
-        <use id="btnicon" xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-close"></use>
+        <use class="btnicon" xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-close"></use>
```

Comment:
Do we think this class is even needed?

Reply:
@Kingdutch, what do you mean `the icon no longer changed`? I think this class no needed in this case because this class does not use for `post photo remove` button

Reply:
I've checked in the code base and we only use the class for this specific button so we don't need it on other buttons (it has no styling and the JavaScript is scoped to this dropdown).

I've updated the code accordingly :)